src/Meshes/linesMesh.ts

```diff
@@ -263,7 +263,17 @@ export class LinesMesh extends Mesh {
      * @returns a new InstancedLinesMesh
      */
     public createInstance(name: string): InstancedLinesMesh {
-        return new InstancedLinesMesh(name, this);
+        let instance = new InstancedLinesMesh(name, this);
+
+        if (this.instancedBuffers) {
+            instance.instancedBuffers = {};
+
+            for (var key in this.instancedBuffers) {
+                instance.instancedBuffers[key] = this.instancedBuffers[key];

```

Comment:
I think you should set it to `null` instead, as it is done in `Mesh.registerInstancedBuffer`. Else the instances will use the same attributes than the master.

Reply:
It is actually a copy paste from instancedMeshFactory :-)

```
Mesh._instancedMeshFactory = (name: string, mesh: Mesh): InstancedMesh => {
    let instance = new InstancedMesh(name, mesh);

    if (mesh.instancedBuffers) {
        instance.instancedBuffers = {};

        for (var key in mesh.instancedBuffers) {
            instance.instancedBuffers[key] = mesh.instancedBuffers[key];
        }
    }

    return instance;
};
```

Reply:
Ok, my mistake, I thought `instancedBuffers` were vertex buffers but they are custom data instead, that can be overwritten afterward by the user!